File: year2020/month10/day1022/ms_1022.cs
using System;

namespace algorithm_84
{
    class ms_1022
    {
        static void Main(string[] args)
        {
            String[] input = Console.ReadLine().Split(' ');
            String data = input[0];
            int range = Convert.ToInt32(input[1]);
            int dl = data.Length;
            int min = range;

            if(dl > range)
                min = dl - range;
            else min = 0;

            if(dl <= 100) 
                for(int i = dl - 1; i >= min; i--) 
                    Console.Write(data[i]);
        }
    }
}

File: year2020/month10/day1022/ms_1022.cpp
#include <iostream>
#include <cstring>
#include <sstream>

using namespace std;

int main() {
	string input;
	
	getline(cin, input);

	stringstream ss(input);
	string data[2];
	int i = 0;

	while (getline(ss, input, ' ')) {
		data[i] = input;
		i++;
	}

	int range = stoi(data[1]);
	int dl = data[0].length();
	int min = range;

	if (dl > range) 
		min = dl - range;
	else min = 0;

	if (dl <= 100) {
		for (i = dl - 1; i >= min; i--) {
			cout << data[0][i];
		}
	}

	return 0;
}